client/client_integration_v0_test.go

```diff
@@ -135,3 +136,34 @@ func TestGetDefaultAccountID(t *testing.T) {
 			defaultAccountId, accountIdFromName)
 	}
 }
+
+// TestQuery with set statements
+func TestQuerySetStatements(t *testing.T) {
+	query := "SELECT '2024-01-01 00:00:00'::timestamptz"
+	response, err := clientMock.Query(
+		context.TODO(),
+		engineUrlMock,
+		query,
+		map[string]string{"time_zone": "America/New_York", "database": databaseMock},
```

Comment:
lets add a comment saying v0 has `time_zone` so we don't try to merge these tests again later on.